Improving the user guide for the Nextcloud integration

wielinde opened the pull request:
- Link to the Nextcloud integeration guide from the user guide root.
- Improve instruction for creating file links.
- Update screenshot for "Connected accounts".

https://community.openproject.org/work_packages/41752

wielinde committed:
[#41752] Documentation updated (docs.openproject.com)

- Link to the Nextcloud integeration guide from the user guide root.
- Improve instruction for creating file links.
- Update screenshot for "Connected accounts".

https://community.openproject.org/work_packages/41752

psatyal committed:
Further changes to phrasing

I added additional details and precisions where I thought they would improve the comprehensabilty of the text, and additionally fixed some typos/grammatical oddities.

psatyal reviewed (approved):
Hi @wielinde, I think your changes make things clearer.

I've added additional details and precisions where I thought they would improve the comprehensabilty of the text, and additionally fixed some typos/grammatical oddities.

@marcalcobe I've left the existing screenshots untouched for the moment; I'll leave making any changes up to your discretion.

marcalcobe committed:
Updated screenshots with list view

I've updated all the screenshots about linking a WP in Nextcloud. Now there is a list view and they are in the correct placement.

marcalcobe commented:
Hi @wielinde and @psatyal,

I've actualised the screenshots and position them in the correct place.
I think the only remaining open discussion would be the order in which the index is displayed. @as-op can you confirm us that is in the correct order?

psatyal committed:
Minor (typo)
